man/signal-cli-dbus.3.adoc

```diff
@@ -0,0 +1,205 @@
+/////
+vim:set ts=4 sw=4 tw=82 noet:
+/////
+:quotes.~:
+
+= signal-cli-dbus (3)
+
+== Name
+
+DBus API for signal-cli - A commandline and dbus interface for the Signal messenger
+
+== Synopsis
+
+*signal-cli* [--config CONFIG] daemon --dbus-system
+
+*dbus-send* --system --type=method_call --print-reply --dest="org.asamk.Signal" /org/asamk/Signal org.asamk.Signal<method> [string:<string argument>] [array:<type>:<array argument>]
+
+
+== Description
+
+See signal-cli (1) for details on the application.
+
+This documentation handles the supported methods when running signal-cli as a DBus daemon.
+
+The method are described as follows:
+
+Return<type> = Method( arg1<type>, arg2<type>, ...)
```

Comment:
stylistic remark, please make sure all methods use the same formatting
- camel case for names (no spaces in argument name)
- no spaces before or after brackets
- a single space after each comma
- maybe put the return type behind the method?

```suggestion
Method(arg1<type>, arg2<type>, ...) → Return<type>
```